NGitLab/Models/Package.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Text.Json.Serialization;
+using NGitLab.Impl.Json;
+
+namespace NGitLab.Models
+{
+    public class Package
+    {
+        [JsonPropertyName("id")]
+        public int Id { get; set; }
```

Comment:
New classes should use `long` for ids

```suggestion
        public long Id { get; set; }
```